bundle/edu.gemini.obslog/src/main/java/edu/gemini/obslog/obslog/DBNightlyPlanInfo.java

```diff
@@ -22,7 +22,7 @@
     // last modified timestamp
     private long _timestamp;
 
-    private static SimpleDateFormat _formatter = new SimpleDateFormat("MM/dd/yy HH:hh:ss");
+    private static SimpleDateFormat _formatter = new SimpleDateFormat("yyyy-MMM-dd HH:hh:ss");
```

Comment:
I know it isn't part of this PR, but a `static SimpleDateFormat` is always a danger sign.  The issue is that it isn't thread-safe.  I think we might as well switch these to the newer thread-safe `java.time.format.DateTimeFormatter` while we're at it.